testsuite/vpi/vpi001/testsuite.sh

```diff
@@ -5,7 +5,7 @@
 analyze mydesign.vhdl
 elab myentity
 
-if ghdl_has_feature myentity vpi; then
+if c_compiler_is_available && ghdl_has_feature myentity vpi; then
```

Comment:
Instead of skipping each vpi test, I would skip the whole (sub)testsuite at the top of https://github.com/LukasVik/ghdl/blob/testsuite_gcc/testsuite/vpi/testsuite.sh